psf-new-code-of-conduct.md

```diff
@@ -69,7 +72,7 @@ Thank you for helping make this a welcoming, friendly community for everyone.
 
 ### PSF Events
 
-This Code of Conduct applies to the following people at events sponsored by the Python Software Foundation:
+This Code of Conduct applies to the following people at events hosted by the Python Software Foundation, and events hosted by projects under the PSF's fiscal sponsorship:
```

Comment:
Hosted is the correct term here to limit scope correct? From our conversations and my memory this does seem correct but want to make sure!